model/qti/attribute/ExternalScored.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ *
+ */
+
+namespace oat\taoQtiItem\model\qti\attribute;
+
+use oat\taoQtiItem\model\qti\attribute\Interpretation;
+use oat\taoQtiItem\model\qti\attribute\Attribute;
+
+/**
+ * The externalScored attribute
+ *
+ * @access public
+ * @author Anshul Sharma, <ansul@taotesting.com>
+ * @package taoQTI
+
+ */
+class ExternalScored extends Attribute
+{
+
```

Comment:
```suggestion
```

Reply:
Not fixing it because we made an agreement this morning to reduce turn around time for a PR by not bothering with spaces and returns. Also, if you check the other attribute PHP files, they have this return too.